lib/collections.js

```diff
@@ -48,6 +67,31 @@ longAndShort['shortest'] = paragraph;
 
 /* your code starts here */
 
+// declare longest, shortest, current longest and shortest word tp keep track of results.
+var longest = 0;
+var shortest = 100;
```

Comment:
It's actually not necessary to store these - you can just keep the shortest and longest words, and just run `.length` on them to get their lengths.